benchmarks/kubectl-mtb/types/types.go

```diff
@@ -14,5 +14,6 @@ type RunOptions struct {
 	Label           string
 	KClient         *kubernetes.Clientset
 	TClient         *kubernetes.Clientset
+	OClient         *kubernetes.Clientset
```

Comment:
Can we be a little bit more descriptive while naming different clients?

Reply:
Yes, agreed. What would be good names here? Something like:
- ClusterAdminClient
- Tenant1Client
- Tenant2Client